app-authz-jakarta-servlet/pom.xml

```diff
@@ -35,55 +35,69 @@
         </license>
     </licenses>
 
+    <properties>
+        <version.keycloak>${project.version}</version.keycloak>
+    </properties>
+
     <dependencies>
         <dependency>
-            <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-core</artifactId>
+            <groupId>jakarta.servlet</groupId>
+            <artifactId>jakarta.servlet-api</artifactId>
+            <version>6.0.0</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-adapter-core</artifactId>
-            <scope>provided</scope>
+            <artifactId>keycloak-policy-enforcer</artifactId>
+            <version>${version.keycloak}</version>
         </dependency>
+
+        <!-- Test Dependencies -->
         <dependency>
             <groupId>org.keycloak</groupId>
-            <artifactId>keycloak-adapter-spi</artifactId>
-            <scope>provided</scope>
+            <artifactId>keycloak-client-registration-api</artifactId>
+            <version>${version.keycloak}</version>
+            <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
-            <version>4.5</version>
-            <scope>compile</scope>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-admin-client-jakarta</artifactId>
+            <version>21.0.2</version>
```

Comment:
I guess the hardcoded version here should be avoided?

Is it possible to use `<version>${version.keycloak}</version>` ? Or ideally avoid `<version>` element entirely and make sure that correct version is just taken from the parent pom (keycloak BOM) as I've mentioned in the other comments to this file?